cloto/manager/AuthorizationManager.py

```diff
@@ -24,11 +24,14 @@
 #
 __author__ = 'gjp'
 from keystoneclient.exceptions import AuthorizationFailure, Unauthorized, InternalServerError
+from keystoneclient.exceptions import ClientException as KeystoneClientException
+from keystoneclient.exceptions import ConnectionRefused as KeystoneConnectionRefused
 from cloto.models import TokenModel
 import json
 import requests
-from cloto.constants import ACCEPT_HEADER, JSON_TYPE, X_AUTH_TOKEN_HEADER, TOKENS_PATH, SERVICE_NOT_AUTORIZED, \
-    TOKEN_NOT_FOUND
+import logging as logger
```

Comment:
This is strange and may lead to confusion. Either you use the default root logger provided by the module (therefore, you use the module-level `logging.info`, `logging.debug`, etc. functions) or you use `getLogger` to create your own logger object. Alternatively, if not comfortable with using the module itself, you could simply set `logger = logging.root`.